examples/simple/main.tf

```diff
@@ -0,0 +1,21 @@
+data "aws_vpc" "default" {
+  default = true
+}
+data "aws_subnet_ids" "default" {
+  vpc_id = data.aws_vpc.default.id
+}
+module "simple" {
+  source = "../../"
+  #source  = "USSBA/apex-redirect/aws"
```

Comment:
should these be commented out?

Reply:
yeah, the convention in terraform module examples is to just do `../..`, but I like having the direct source/version lines included for ease of copy/paste